src/libs/actions/App.js

```diff
@@ -40,6 +46,20 @@ function setLocale(locale) {
     Onyx.merge(ONYXKEYS.NVP_PREFERRED_LOCALE, locale);
 }
 
+function getLocaleAndUpdate() {
```

Comment:
not a blocker - The function name sounds a little odd to me, like something was missing after "and update".  Maybe `loadLocale()` would be an alternative.

Reply:
We are using `get*` or `fetch*` in a lot of methods so probably something like `getLocale()` is fine here?

https://github.com/Expensify/App/blob/b770151b8c4f384244f847258a18657629b2bb49/src/libs/Navigation/AppNavigator/AuthScreens.js#L155-L157